v0/destinations/am/transform.js

```diff
@@ -274,7 +277,10 @@ function responseBuilderSimple(
       traits = getFieldValueFromMessage(message, "traits");
       set(rawPayload, "event_properties", message.properties);
       if (traits) {
-        set(rawPayload, "user_properties", traits);
+        rawPayload.user_properties = {
+          ...rawPayload.user_properties,
+          ...traits
```

Comment:
There could be conflict between `context.traits.name` and `campaign.name`

Reply:
I think we can prepend the keys with `utm_`

Reply:
Yeah definitely that would cause a conflict. So will append the campaign keys with utm_

Reply:
Added those changes now